src/character.cpp

```diff
@@ -509,6 +509,15 @@ character_id Character::getID() const
     return this->id;
 }
 
+void Character::randomize_height()
+{
+    // Height distribution data is taken from this paper:
+    // http://www.biostat.jhsph.edu/bstcourse/bio751/papers/bimodalHeight.pdf
+    const double x = male ? normal_roll( 176.0, 7.4 ) : normal_roll( 162.8, 7.0 );
+    // clamping to 145..200 because this is the bounds of what player can set, see newplayer.cpp
+    init_height = clamp( static_cast< int >( round( x ) ), 145, 200 );
```

Comment:
```suggestion
    init_height = clamp( static_cast< int >( std::round( x ) ), 145, 200 );
```

I think this is what lgtm wants to see.

Reply:
Thank you for cooperation